experiments/verify_fasm_to_bels.yaml

```diff
@@ -1,7 +1,7 @@
 designs:
+    - base/
```

Comment:
I don't know if everyone in the base folder compares as equivalent.  Does it?  Let's only include things that come back as equivalent.